implementation-jaxrs2/src/main/java/io/smallrye/opentelemetry/jaxrs2/client/ClientSpanDecorator.java

```diff
@@ -0,0 +1,52 @@
+package io.smallrye.opentelemetry.jaxrs2.client;
+
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientResponseContext;
+
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.attributes.SemanticAttributes;
+import io.smallrye.opentelemetry.jaxrs2.server.internal.URIUtils;
+
+/**
+ * @author Pavol Loffay, Felix Wong
+ */
+public interface ClientSpanDecorator {
+
+    /**
+     * Decorate get by incoming object.
+     *
+     * @param requestContext
+     * @param span
+     */
+    void decorateRequest(ClientRequestContext requestContext, Span span);
+
+    /**
+     * Decorate spans by outgoing object.
+     *
+     * @param responseContext
+     * @param span
+     */
+    void decorateResponse(ClientResponseContext responseContext, Span span);
+
+    /**
+     * Adds standard tags: {@link io.opentracing.tag.Tags#SPAN_KIND},
+     * {@link io.opentracing.tag.Tags#PEER_HOSTNAME}, {@link io.opentracing.tag.Tags#PEER_PORT},
```

Comment:
* update the comment to match the semantic attributes. 
* the peer hostname nor port is being set